fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -118,8 +118,16 @@ fun OnboardingPage(
 
                 DescriptionText(
                     description = pageState.description,
-                    linkTextState = pageState.linkTextState,
                 )
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                pageState.privacyCaption?.apply {
```

Comment:
🔍 `let` or an `if` would suit this case better. [apply](https://kotlinlang.org/docs/scope-functions.html#apply) is good for builder pattern objects for configuration

Reply:
Sure, I'll use let.  Thanks